query-creator/create-query/app.js

```diff
@@ -19,7 +19,7 @@ exports.lambdaHandler = async (event, context) => {
     const tokenServiceEnv = params.tokenServiceEnv;
     const usageReport = params.usageReport || false;
     const useLogs = params.useLogs || false;
-    const endpointOnly = params.endpointOnly || false
+    const narrowLearners = params.narrowLearners || false
```

Comment:
Briefer option for accomplishing the same thing:
```typescript
  const { reportServiceSource, debugSQL = false, tokenServiceEnv, usageReport = false, useLogs = false, narrowLearners = false } = params;
```

Reply:
the more concise version would behave slightly differently: if one of these params was the empty string `""` the current code would turn that into false. The concise version would leave the value `""`. 

I don't know if that would matter in this case.

I'm going to merge this as is, but feel free to make it more concise if it makes sense.